app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,279 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
+  extend ActiveSupport::Concern
+
+  included do
+    def dummy_client
+      @dummy_client ||= ExTwitter::Client.new
+    end
+
+    def one_sided_following
+      dummy_client.one_sided_following(self)
+    end
+
+    def one_sided_followers
+      dummy_client.one_sided_followers(self)
+    end
+
+    def mutual_friends
+      dummy_client.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      dummy_client.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      dummy_client.common_followers(self, other)
+    end
+
+    def removing
+      return [] unless TwitterUser.has_more_than_two_records?(uid, user_id)
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
```

Comment:
__[rubocop]__
* Line is too long. [114/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)